src/App.jsx

```diff
@@ -1,27 +1,72 @@
+import { setUtahHeaderSettings } from '@utahdts/utah-design-system-header';
+import { useEffect } from 'react';
 import { useRemoteConfigString } from 'reactfire';
-import './App.css';
-import { fieldNames } from './config';
+import MapComponent from './components/Map.jsx';
+import SearchWizard from './components/search-wizard/Wizard.jsx';
 
 function App() {
   const queryLayersConfig = useRemoteConfigString('queryLayers');
+  const linksConfig = useRemoteConfigString('links');
+
+  useEffect(() => {
+    if (linksConfig.status !== 'success') {
+      return;
+    }
+
+    const links = JSON.parse(linksConfig.data);
+    setUtahHeaderSettings({
+      // this prop's implementation has not been released yet
+      // domLocationTarget: {
+      //   element: headerRef.current,
+      // },
+      title: 'Utah Environmental Interactive Map',
+      titleURL: null,
+      logo: '<img src="/deq_logo.png" alt="DEQ Logo" />',
+      utahId: false,
+      actionItems: [
+        {
+          actionPopupMenu: {
+            menuItems: [
+              {
+                actionUrl: {
+                  url: links['2'].url,
+                  openInNewTab: true,
+                },
+                title: links['2'].description,
+              },
+              {
+                // TODO: add a modal for this
+                actionFunction: () => window.alert('not implemented yet!'),
+                title: 'Disclaimer',
+              },
+            ],
+            title: 'Links',
+          },
+          className: 'icon-waffle',
+          showTitle: false,
+          title: 'Links',
+          icon: '<span class="utds-icon-before-waffle" aria-hidden="true" />',
+        },
+      ],
+    });
+  }, [linksConfig]);
 
   if (queryLayersConfig.status === 'loading') {
     console.log('loading remote config');
```

Comment:
**suggestion**: consider using a custom logger so you can choose when to write these messages since the build doesn't strip them.

Reply:
No not gcp. Just the console. Like if env != production then console.log type of deal with otherwise nada.

Reply:
I don't mind seeing limited logs in prod. That can be helpful in diagnosing problems.

Reply:
Sure. I thought we had an implementation in one of our projects that was like

```js
const x = Log()
x.log('debug') // removed in prod
x.log('helpful', {force: true}); // kept in prod
``` 

No big deal, it was just a suggestion.

That makes me think to suggest adding firebase analytics and log events so we can tell how the app is being used.

```js
import { getAnalytics, logEvent } from 'firebase/analytics';
```